test/boost/conftest.py

```diff
@@ -16,6 +17,9 @@
 from test.pylib.cpp.common_cpp_conftest import collect_items, get_combined_tests
 from test.pylib.util import get_modes_to_run
 
+def pytest_addoption(parser):
+    parser.addoption("--gather-metrics", action=argparse.BooleanOptionalAction, default=True)
```

Comment:
why only boost?

Reply:
Work in progress. When it is fully ready for boost, the rest will have it as well.